examples/BodyPose-blazepose-skeleton/index.html

```diff
@@ -0,0 +1,19 @@
+<!--
+ Copyright (c) 2024 ml5
+ 
+ This software is released under the MIT License.
+ https://opensource.org/licenses/MIT
+-->
+
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>ml5.js BodyPose BlazePose Skeleton Example</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.9.0/p5.js"></script>
```

Comment:
```suggestion
    <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.9.2/p5.js"></script>
```

let's try to keep everything updated to latest p5.js (esp due to the mirroring feature)

Reply:
Yes, we should also update these on all other examples!